packages/core/src/Modules/Cashier/Components/confirm.jsx

```diff
@@ -16,7 +17,13 @@ const Row = ({ label, value }) => (
                 ))}
             </span>
         ) : (
-            <strong>{value}</strong>
+            <strong
+                className={classNames({
+                    desc: label === 'Description',
```

Comment:
Won't work for localized strings I think?

Reply:
Oh good point :+1: